src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,20 @@
+<div>
+    <p>When you’re ready to apply we need two things:</p>
+    <ol>
+      <li>1. A completed questionare
```

Comment:
Since we're using an `ol` here, we don't need to manually put the numbers in. They'll be there. Screen readers would also read "1., 1." "2., 2.".

Reply:
fixed!